tools/ci_build/github/apple/default_full_apple_framework_build_settings.json

```diff
@@ -19,6 +19,7 @@
             "--build_apple_framework",
             "--use_coreml",
             "--use_xnnpack",
+            "--use_webgpu",
```

Comment:
we shouldn't enable this in the packaging configs until the full working version is in
```suggestion
```